<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spbMainWindow</class>
 <widget class="QMainWindow" name="spbMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>spbMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <item>
       <layout class="QVBoxLayout" name="vlCommandSettings" stretch="0,0,1,0,0">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Command code</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Interval</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="sbInterval">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>16.384000000000000</double>
            </property>
            <property name="maximum">
             <double>4194.304000000000087</double>
            </property>
            <property name="singleStep">
             <double>16.384000000000000</double>
            </property>
            <property name="value">
             <double>16.384000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="hlCCode">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QSpinBox" name="sbCCode">
              <property name="minimum">
               <number>16</number>
              </property>
              <property name="maximum">
               <number>254</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="bCCodeHex">
              <property name="toolTip">
               <string>View command code as hex value</string>
              </property>
              <property name="text">
               <string>FF</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Steps:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="bCmdPosAdd">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="bCmdPosDel">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="bCmdPosUp">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="bCmdPosDown">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="lwCmdStepList">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>210</width>
            <height>16777213</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Foots:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lwFootList">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>210</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vlViewer"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QToolButton" name="bPalyPause">
        <property name="toolTip">
         <string>Play</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="bStepPlay">
        <property name="toolTip">
         <string>Step play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="bStop">
        <property name="toolTip">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="bRepeat">
        <property name="toolTip">
         <string>Repeat</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sTiming">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lPosition">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>1/1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_project"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionFootEditor"/>
   </widget>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionAdd_new_command"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuCommands"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_project">
   <property name="text">
    <string>Open project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>Save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFootEditor">
   <property name="text">
    <string>FootEditor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAdd_new_command">
   <property name="text">
    <string>Add new command</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
